BraveRewards/Ledger/BATBraveLedger.mm

```diff
@@ -552,6 +613,22 @@ - (void)onReconcileComplete:(ledger::Result)result viewingId:(const std::string
                                        nowTimestamp);
   }
   
+  if ((result == ledger::Result::LEDGER_OK && category == ledger::REWARDS_CATEGORY::AUTO_CONTRIBUTE) ||
+      result == ledger::Result::LEDGER_ERROR ||
```

Comment:
Do we want to add notification even if `result` has error states?

Reply:
Logic was copied from brave-core's implementation

Reply:
Yeah, so as seen here: https://github.com/brave/brave-core/blob/f6956d15bbc9ccf51faceafe5d74c4c3f4e525b2/components/brave_rewards/browser/rewards_notification_service_impl.cc#L348

The result is passed into the notification info, so I assume when we go to implement the UI shown for this notification (#45), it will change based on what the result here is. Since this is actually going to happen in the UI, I'm going to change this to not be a string